features/uri-templating.feature

```diff
@@ -65,6 +66,15 @@ Functionaliteit: URI templating
     | href                                            | templated |
     | /kadasternietnatuurlijkpersonen/{identificatie} | true      |
 
+  Scenario: Verwijzingen naar een Resource type via meerdere properties van één of meerdere gegevensgroepen
+    Gegeven een ZakelijkGerechtigde heeft een verwijzing via de 'stukIdentificaties' property van gegevensgroep 'ZakelijkRecht' naar Stukken
+    En de ZakelijkGerechtigde heeft een vewijzing via de 'isVermeldInStukdeelIdentificaties' property van de gegevensgroep 'ZakelijkRecht' naar Stukdelen
+    En de ZakelijkGerechtigde heeft een verwijzing via de 'stukIdentificaties' property van de gegevensgroep 'Tenaamstelling' naar Stukken
+    En de ZakelijkGerechtigde heeft een vewijzing via de 'isVermeldInStukdeelIdentificaties' property van de gegevensgroep 'Tenaamstelling' naar Stukdelen
+    Als de links voor de ZakelijkGerechtigde is gegenereerd
+    Dan bevat de _links gegevensgroep van de ZakelijkGerechtigde een 'Stukken' property waarmee een link naar een Stuk kan worden samengesteld voor zowel de 'ZakelijkRecht' en 'Tenaamstelling' gegevensgroep
+    En bevat de _links gegevensgroep van de ZakelijkGerechtigde een 'Stukdelen' property waarmee een link naar een Stukdeel kan worden samengesteld voor zowel de 'ZakelijkRecht' en 'Tenaamstelling' gegevensgroep
```

Comment:
is het niet nog duidelijker wanneer er echt een voorbeeld uit een response bij zit?
Bijvoorbeeld zoiets als:
En de ZakelijkGerechtigde heeft property tenaamstelling.stukIdentificatie="123456"
En de ZakelijkGerechtigde heeft property zakelijkRecht.stukIdentificatie="789012"
En de ZakelijkGerechtigde heeft property _links.stukken[0].href="/stukken/{stukIdentificatie}"
Dan kan het stuk voor tenaamstelling worden geraadpleegd op uri "/stukken/123456"
En kan het stuk voor zakelijkRecht worden geraadpleegd op uri "/stukken/789012"